### ISSUE TITLE
Math in LLMs

### ISSUE THREAD

**Issue by fivetwentythree** [fivetwentythree/wiki#17]
![The maths you need to start understanding LLMs](https://www.gilesthomas.com/images/favicons/web-app-manifest-512x512.png)

A quick refresher on the maths behind LLMs: vectors, matrices, projections, embeddings, logits and softmax.

[Source: The maths you need to start understanding LLMs via Giles' Blog](https://www.gilesthomas.com/2025/09/maths-for-llms)
